system/include/libc/features.h

```diff
@@ -33,4 +33,7 @@
 #define _Noreturn
 #endif
 
+#define weak_alias(old, new) \
+	extern __typeof(old) new __attribute__((__weak__, __alias__(#old)))
```

Comment:
This looks like its already defined in `src/internal/libc.h`..  And in upstream musl it moved to the internal header  `src/include/features.h` (as opposed to this features.h header which is externally visible).

Is this a necessary part of the change?  Do the sources in string not already include `src/internal/libc.h`?

I wonder if its worth bringing over the new `src/include/features.h` wrapper header?

Reply:
Well, you nerd-sniped me. I am seeing how much v1.2.0 I can pull in here.

Reply:
If possible I'd recommend keeping this change focused, unless you want to do a full musl upgrade .. which is quite a big endeavor.  (I think it took weeks or maybe months to completely last time it was done).

Reply:
I agree, let's land this now, but @zekoz please submit more PRs with more 1.2 pieces or even the whole thing upgrade!